src/test/java/com/cgdp/library/validation/ValidatorTest.java

```diff
@@ -0,0 +1,104 @@
+package com.cgdp.library.validation;
+
+import com.cgdp.library.exceptions.ValidationException;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.catchThrowable;
+
+public class ValidatorTest {
+
+    @Test
+    void should_return_null_if_value_is_null() {
```

Comment:
Naming is inaccurate, it is not returning anything, it is throwing an exception
```suggestion
    void should_throw_exception_when_value_is_null() {
```
Do the same thing for other tests